doc/ChangeLog

```diff
@@ -1,4 +1,81 @@
 ===============================================================
+Tag name:  ctsm5.1.dev053
```

Comment:
ChangeLog and ChangeSum show up here at this time because I merged the `hist_vars_for_newton_krylov` branch to this branch. They will disappear from here when the `hist_vars_for_newton_krylov` branch gets merged to master.

Reply:
Not true since I removed my mods to ChangeLog and ChangeSum when I merged with dev061 today.